dpe/src/dpe_instance.rs

```diff
@@ -239,6 +239,17 @@ impl<C: Crypto> DpeInstance<'_, C> {
         Ok(())
     }
 
+    fn serialize_internal_dpe_info(&self, internal_dpe_info: &mut [u8; INTERNAL_DPE_INFO_SIZE]) {
+        // Internal DPE Info contains get profile response fields as well as the size of the hashing algorithm used
+        let profile_size = self
+            .get_profile()
+            .unwrap()
+            .serialize(internal_dpe_info)
+            .unwrap();
+        internal_dpe_info[profile_size..]
+            .copy_from_slice(&DPE_PROFILE.get_hash_size().to_le_bytes());
```

Comment:
minor: I think I would prefer to include the profile ID instead of the hash size in case we have a future profile id with the same hash size (I think that is likely).

Should be able to do `(DPE_PROFILE as u32).to_le_bytes()`